src/beacon-api/main.py

```
@@ -111,16 +108,20 @@ async def global_exception_handler(request: Request, exc: Exception):
 @app.get("/")
 def root():
     return {"message": f"Welcome to {settings.APP_NAME}"}
 
-@app.get("/health")
-def health():
-    return {"status": "ok"}
-
 if __name__ == "__main__":
-    port = int(os.environ.get("PORT", 8000))
-    uvicorn.run("main:app", host="0.0.0.0", port=port, reload=True)
+    uvicorn.run("main:app", host="0.0.0.0", port=8000, reload=True)
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)